cloudify_gcp/compute/instance.py

```diff
@@ -189,6 +189,7 @@ def get(self):
             project=self.project,
             zone=basename(self.zone)).execute()
 
+    @utils.sync_operation
```

Comment:
this makes the function actually wait for the response: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/blob/5a09f4e740dd4837929bfce26e0435f3000dcba2/cloudify_gcp/utils.py#L143-L152